launcher/src/local-storage-key-provider.ts

```diff
@@ -28,6 +28,7 @@ export class LocalStorageKeyProvider {
       console.log(`  > found key file ${publicKeyFile}`);
 
       const secret = await this.getPartOfPublicKey(publicKeyFile);
+      console.log(`  > secret: ${secret}`);
```

Comment:
I think it is enough to add a test for that and do not show the secret.
From the other side, I do not mind to keep it here, you can just wrap the secret in quotes to understand whether it contains a newline character.